javascript-light-bulb/main.js

```diff
@@ -0,0 +1,19 @@
+var onOff = 0;
+
+function handleLight(event) {
+  onOff++;
+  var change = '';
+  if (onOff % 2 === 1) {
```

Comment:
Instead of using a counter, I would use a Boolean value that you can toggle to `true` or `false`